plugins/mcreator-core/blockly/mcreator_extensions.js

```diff
@@ -93,4 +93,90 @@ Blockly.Extensions.registerMutator('variable_entity_input',
                 this.removeInput('entity');
             }
         }
-    });
\ No newline at end of file
+    });
+
+// Extension used by int providers to validate their min/max values, so that min can't be greater than max and vice versa
+Blockly.Extensions.register('min_max_fields_validator',
+    function() {
+        var minField = this.getField('min');
+        var maxField = this.getField('max');
+
+        // If min > max, we set its value to that of max
+        minField.setValidator(function(newValue) {
+            if (newValue > maxField.getValue()) {
+                return maxField.getValue();
+            }
+            return newValue;
+        });
+
+        // If max < min, we set its value to that of min
+        maxField.setValidator(function(newValue) {
+            if (newValue < minField.getValue()) {
+                return minField.getValue();
+            }
+            return newValue;
+        });
+    });

```

Comment:
This line should not be indented

Reply:
Yes, it should be, as the `function` word is at the same indentation.

Reply:
); belongs to the register call that is not indented. It is same as e.g.:

![image](https://user-images.githubusercontent.com/16374228/210859099-a6cdd784-5afa-468e-893f-fa3d68950258.png)

Reply:
If there was something longer than `generatorFiles.forEach()`, it would be the same thing.

Reply:
https://github.com/MCreator/MCreator/blob/8dc8a786c716d7a100978346bf0042a70a85134a/src/main/java/net/mcreator/ui/modgui/codeviewer/ModElementCodeViewer.java#L73-L104

Reply:
![image](https://user-images.githubusercontent.com/16374228/210859721-64364298-a697-4c3d-ba84-4dedf1726aad.png)

Reply:
![image](https://user-images.githubusercontent.com/16374228/210859846-2f40ba09-2264-455f-ad8c-e6279b5a92a3.png)

Reply:
You're not pointing at the same thing at those two screenshots. In Java, you're pointing at the line where the lambda begins; in Blockly, you're pointing at the beginning of the statement, but not at the line where the function begins.

Reply:
} belongs to where the function begins, but ); belongs to where the register call begins in a non-indented line. I leave this to SomeoneElse to decide what he prefers

Reply:
This indentation is consistent with the other extensions, so I think we can keep it as is